.github/workflows/windows_conditional_compilation.yml

```diff
@@ -109,7 +109,10 @@ jobs:
           version: "v0.5.4"
```

Comment:
interesting why sccache version is different in Windows CC..

Reply:
Fixed.